packages/squiggle-lang/src/rescript/Reducer/Reducer_Peggy/Reducer_Peggy_ToExpression.res

```diff
@@ -17,14 +17,35 @@ let rec fromNode = (node: Parse.node): expression => {
     ExpressionBuilder.eFunction("$$_lambda_$$", list{args, body})
   }
 
+  // Convert function calls to macro calls
+  let caseCallIdentifier = (nodeCallIdentifier: Parse.nodeCallIdentifier): expression => {
+    let callIdentifier = nodeCallIdentifier["value"]
+    // `callIdentifier ${callIdentifier}`->Js.log
+    let callValue = switch callIdentifier {
+    | "withEnvironmentSampleCount" => "$$_withEnvironmentSampleCount_$$"
+    | callName => callName
+    }
+    ExpressionBuilder.eCall(callValue)
+  }
+
+  // Convert identifiers to macro calls
+  let caseIdentifier = (nodeIdentifier: Parse.nodeIdentifier): expression => {
+    let identifier = nodeIdentifier["value"]
+    // `caseIdentifier ${identifier}`->Js.log
+    switch identifier {
+    | "environment" => "$$_environment_$$"->ExpressionBuilder.eFunction(list{})
```

Comment:
Why not call this `Environment` (capitalized), and treat it like a module, similar to `System` and `Danger`?

Reply:
@OAGr 
Just Environment or System.Environment?
e.g.
System.Environment.sampleCount
vs
Environment.sampleCount

Reply:
I think the latter; just `Environment.sampleCount`.